README.md

```diff
@@ -14,13 +14,26 @@ Clone a stable version of the repository using Git by entering a command similar
 
 where \<tag\> is stable version number, like **2022.3.0**.
 
-Alternatively, you can download a zip file containing a stable version of the repository. 
+Alternatively, you can download a zip file containing a stable version of the repository.
 
 1. Select the appropriate tag.
 2. Click the **Code** button.
 3. Select **Download ZIP** from the menu options.
 4. After downloading the file, unzip the repository contents.
 
+### Getting Started with oneAPI Samples
+
+The best oneAPI sample to start with depends on what you are trying to learn or types of problems you are trying to solve.
+
+| If you want to learn about...                           | Start with...
+|:---                                                     |:---
+| the basics of writing, compiling, and building programs for CPUs, GPUs, or FPGAs <br> |[Simple Add](https://github.com/oneapi-src/oneAPI-samples/tree/master/DirectProgramming/DPC%2B%2B/DenseLinearAlgebra/simple-add) or  [Vector Add](https://github.com/oneapi-src/oneAPI-samples/tree/master/DirectProgramming/DPC%2B%2B/DenseLinearAlgebra/vector-add) samples <br> (You can use these samples as starter projects by removing unwanted elements and adding your code and build requirements.)
```

Comment:
```suggestion
| the basics of writing, compiling, and building programs for CPUs, GPUs, or FPGAs <br> |[Simple Add](DirectProgramming/DPC++/DenseLinearAlgebra/simple-add) or  [Vector Add](/DirectProgramming/DPC++/DenseLinearAlgebra/vector-add) samples <br> (You can use these samples as starter projects by removing unwanted elements and adding your code and build requirements.)
```

Reply:
The big advantage of doing it that way it that it removes the branch name, and therefore if someone is using a 20xx.x release, he won't get the **master** version which may not work with his compiler.
Here GitHub redirects to the correct folder in the same branch has the one being in use.